### PR TITLE
Aha/expire process logs

### PR DESCRIPTION
**Pull request by andyHa**

### CODE REVIEW

File: src/main/java/sirius/biz/process/DeleteExpiredProcessesTask.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    }
+
+    @Override
+    public void runTimer() throws Exception {
+        if (elastic != null && elastic.getReadyFuture().isCompleted() && !Sirius.isStartedAsTest()) {
+            tasks.defaultExecutor().fork(this::clenaup);
+        }
+    }
+
+    private void clenaup() {
```
Comment (sabieber) [inline src/main/java/sirius/biz/process/DeleteExpiredProcessesTask.java]: clenaup -> cleanup

File: src/main/java/sirius/biz/process/Process.java [hunk 2]
```diff
@@ -196,6 +197,21 @@
...
+     * Contains the date when the process will be deleted.
+     */
+    public static final Mapping EXPIRES = Mapping.named("expires");
+    @NullAllowed
+    private LocalDate expires;
+
+    /**
+     * Contains period for which this process will be kept.
+     * <p>
+     * For {@link ProcessState#STANDBY standby} processes the denotes the duration for which logs will be kept.
```
Comment (sabieber) [inline src/main/java/sirius/biz/process/Process.java]: the -> this

File: src/main/resources/component-biz.conf [hunk 3]
```diff
@@ -341,6 +342,9 @@ timer.daily {
     # Determines when analytical tasks are scheduled...
     analytical-engine = 23
 
+    # Determines when expires process logs are removed...
+    cleanup-processes = 3
```
Comment (sabieber) [inline src/main/resources/component-biz.conf]: This collides with the hour ES Dumps are created in OXOMI. Would changing this to 2 or 4 be okay? Else we would have to remeber changing this in OX when upgrading the biz version

File: src/main/java/sirius/biz/process/DeleteProcessesTask.java [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+import sirius.biz.tenants.deletion.DeleteTenantTask;
+import sirius.db.es.Elastic;
+import sirius.kernel.commons.Watch;
+import sirius.kernel.di.std.Part;
+import sirius.kernel.di.std.Priorized;
+import sirius.kernel.di.std.Register;
+import sirius.kernel.nls.NLS;
+
+/**
+ * Deletes all processes for a gicen tenant.
```
Comment (sabieber) [inline src/main/java/sirius/biz/process/DeleteProcessesTask.java]: gicen -> given

File: src/main/resources/biz_de.properties [hunk 5]
```diff
@@ -568,6 +568,18 @@ VirtualFile.noDirectory = '${file}' ist kein Verzeichnis.
 VirtualFile.noFile = '${file}' ist keine Datei.
 VirtualFile.cannotResolveChild = Der Pfad '${child}' kann nicht in '${path}' ausgelöst werden.
 VirtualFileSystem.invalidPath = Der Pfad ist ungültig und kann nicht aufgelöst werden: ${path}
+VFSController.actions = Aktionen
```
Comment (sabieber) [inline src/main/resources/biz_de.properties]: Is this really part of this PR?

File: src/main/java/sirius/biz/process/DeleteExpiredProcessesTask.java [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+        elastic.select(Process.class)
+               .eq(Process.STATE, ProcessState.TERMINATED)
```
Comment (welzfabian) [inline src/main/java/sirius/biz/process/DeleteExpiredProcessesTask.java]: what about `CANCELED`?

Comment (andyHa) [reply src/main/java/sirius/biz/process/DeleteExpiredProcessesTask.java]: Every process should eventually become terminated....

**Pull request merged**